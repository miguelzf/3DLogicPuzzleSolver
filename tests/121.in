- - P - -
Y - - - -
G - - - -
B - - - -
L - - - L

- - - - -
- - - - -
- - P - -
- - - - -
- - - - -

R - - - -
- - G - -
- - - - -
- - - Y -
R B - - -
