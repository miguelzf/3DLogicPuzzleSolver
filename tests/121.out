
Solution:
-  -  P  p  p  
Y  y  y  y  y  
G  g  g  g  g  
B  b  b  b  b  
L  l  l  l  L  
=============
r  r  r  r  r  ||  R  b  g  y  p  
r  b  b  b  b  ||  b  b  G  y  p  
r  b  P  p  p  ||  p  p  p  y  p  
r  b  b  b  b  ||  b  b  p  Y  p  
r  r  r  r  r  ||  R  B  p  p  p  
